test/unit/dom/wait_for_test.dart

```diff
@@ -201,6 +205,15 @@ void main() {
                 hasToStringValue(contains('The callback must return a non-null Element.')),
               )));
         }, timeout: asyncQueryTestTimeout);
+
+        test('an element that is never removed, throws with default timeout value', () async {
```

Comment:
Alrighty folks, this test should do the trick. As @sydneyjodon-wk suggested, I'm testing against an element that is never removed. Fails without the default value set (`Cannot read properties of null`), passes when default is set 👍

I removed my previous tests as it looks like the test on line 174 already handles that case.